src/main/java/org/jasig/portlet/newsreader/adapter/RomeAdapter.java

```diff
@@ -132,6 +138,15 @@ public void setProcessor(RomeNewsProcessorImpl processor) {
         this.processor = processor;
     }
 
+    /**
+     * Cache prefix to support multiple instances of this class using the same cache.
+     *
+     * @param prefix    Unique prefix for cache entries for this particular instance
+     */
+    public void setCacheKeyPrefix(String prefix) {
+        this.cacheKeyPrefix = prefix;
+    }
```

Comment:
I don't see @Required on this method -- does that mean it has an effective default of null?